docs/scraping/overview.md

```diff
@@ -164,8 +164,9 @@ metrics:
     azureMetricConfiguration:
       metricName: ActiveMessages
       limit: 5
-      dimension:
-        name: <dimension-name>
```

Comment:
Let's keep this in there as it is still supported, but let's suffix it with `# Deprecated, please use dimensions instead`